### PR TITLE
Make stability attributes an error. #22830

### PR DESCRIPTION
**Pull request by brson**

### CODE REVIEW

File: src/test/compile-fail/stability-attribute-non-staged.rs [hunk 1]
```diff
@@ -8,11 +8,7 @@
...
 // except according to those terms.
 
-// These two generate an error to satisfy the compile-fail test
-#![deny(warnings)]
-#![feature(blah)] //~ ERROR
-
-#[unstable] //~ WARNING: stability attributes are deprecated
-#[stable] //~ WARNING: stability attributes are deprecated
-#[deprecated] //~ WARNING: stability attributes are deprecated
+#[unstable] //~ WARNING: stability attributes may not be used
```
Comment (sfackler) [inline src/test/compile-fail/stability-attribute-non-staged.rs]: WARNING should be ERROR here, right?

Comment (brson) [inline src/test/compile-fail/stability-attribute-non-staged.rs]: Yes, fixed.

**Comment by brson** [comment]
(breaking change)

**Comment by pnkfelix** [comment]
going from nominated to (nominated, accepted), asap.

**Pull request merged**